tests/test_opendss_ingestor.py

```diff
@@ -0,0 +1,41 @@
+import os
+
+import btrdb
+from btrdb.utils.timez import to_nanoseconds
+
+from btrdbextras.opendss_ingest.opendss_ingestor import (
+    MODEL_REL_PATH,initialize_simulation,run_simulation
+)
+
+
+class TestOpendssIngestor:
+    def test_initialize_simulation(self):
+        # Arrange
+        mock_model_loc = os.path.join(
+            MODEL_REL_PATH,
+            "Models/13Bus/IEEE13Nodeckt.dss"
+            )
+        load, load_names = ([1155., 160., 120., 120., 170., 230., 170.,
+                             485., 68., 290., 170., 128., 17., 66., 117.],
+                            ['671', '634a', '634b', '634c', '645', '646', '692',
+                             '675a', '675b', '675c', '611', '652', '670a', '670b',
+                             '670c'])
+        # Act
+        results = initialize_simulation(mock_model_loc)
+        assert results[0].tolist() == load
+        assert results[-1] == load_names
```

Comment:
Nice tests. This setup is great and it'd be useful [in a fixture to improve our testing suite](https://docs.pytest.org/en/latest/how-to/unittest.html#using-autouse-fixtures-and-accessing-other-fixtures). IIUC, we have to run `test_initialize_simulation` before we run `test_simulate_network` otherwise the `dss` model won't be loaded.

To that end, we should make the initialization a fixture both tests can explicitly depend on the setup.

```suggestion
@pytest.fixture(scope="session")
def _initialize_simulation():


class TestOpendssIngestor:
    @pytest.fixutre(scope="session")
    def simulation_setup(self, initialize_simulation):
        # Arrange
        mock_model_loc = os.path.join(
            MODEL_REL_PATH,
            "Models/13Bus/IEEE13Nodeckt.dss"
            )
        # Act
        return initialize_simulation(mock_model_loc)

    def test_initialize_simulation(self, simulation_setup):
            load, load_names = ([1155., 160., 120., 120., 170., 230., 170.,
                             485., 68., 290., 170., 128., 17., 66., 117.],
                            ['671', '634a', '634b', '634c', '645', '646', '692',
                             '675a', '675b', '675c', '611', '652', '670a', '670b',
                             '670c'])
        assert simulation_setup[0].tolist() == load
        assert simulation_setup[-1] == load_names
```

It may also make sense have `run_simulation` use an already initialized test. That way we can test these two functions more easily. So maybe the main function looks more like

```python
def main(stuff):
    results = initialize_simulation(path)
    run_simulation(other_args_plus, results)
```

Reply:
still needs more test added definitely. and the initialize_simulation should be called. I wanted this test to make sure that we always have the Model files. all 3 files are needed, or else we get an error in the subsequent calls to the Opendss modeling code.